leases/models.py

```diff
@@ -183,6 +188,12 @@ class WinterStorageLease(AbstractLease):
         null=True,
         related_name="lease",
     )
+    start_date = models.DateField(
```

Comment:
we could also maybe delete the fields from the abstract base model? now that every actual model has its own implementation